pkg/hcl_interpreter/hcl_interpreter.go

```diff
@@ -245,6 +245,16 @@ func (v *Evaluation) evaluate() error {
 
 		vars := v.prepareVariables(name, term)
 		vars = MergeVal(vars, NestVal(LocalName{"path", "module"}, cty.StringVal(moduleMeta.Dir)))
+		vars = MergeVal(vars, NestVal(LocalName{"path", "root"}, cty.StringVal(moduleMeta.Dir)))
```

Comment:
TODO: need to implement actual root module support, without implicitly assuming that the passed-in module is the root module.